src/ConfigValidator.php

```diff
@@ -176,7 +176,7 @@ private function validateConfig(): void
             $this->logger->info('Loading files with tags ' . var_export($this->input['files'][0]['tags'], true));
         }
         $this->script = implode('\n', $rowData['configuration']['queries']);
-        $this->logger->info('Script size ' . strlen($this->script));
+        $this->logger->info(sprintf('Loaded transformation script (size %s).', strlen($this->script)));
```

Comment:
aha, sorry I didn't see this before, it's chars! (I was wondering if it was bytes).  Still not sure how helpful this is to the user,  how about a `count($rowData['configuration']['queries'])` as number of queries?  Sorry, it's not really important.

Reply:
it's not particularly helpful, but allows to distinguish from empty script and validate that it's roughly what it's supposed to be